src/vs/editor/common/core/selection.ts

```diff
@@ -145,6 +151,13 @@ export class Selection extends Range {
 		return new Selection(this.endLineNumber, this.endColumn, startLineNumber, startColumn);
 	}
 
+	/**
+	 * Create a `Selection` from one positions (not IPosition) to mimic Position.
+	 */
+	public toPositions(start: Position): Selection {
+		return new Selection(start.lineNumber, start.column, start.lineNumber, start.column);
```

Comment:
Should this be static and called `fromPosition` instead?

Reply:
Also since this is in core, think about if there's enough other code in the codebase that could benefit from this new function

Reply:
just removing this entirely to create a `new Selection(...)` elsewhere instead of making the function. Intuitively, it most likely will not be used elsewhere since I'm creating a selection from a single position.